### DISCUSSION TITLE
How to create window not have borderless

### DISCUSSION THREAD

**Discussion by zSakuraEvilz** [AvaloniaUI/Avalonia#14661] [Q&A]
I re write my wpf game laucher to Avalonia.
I couldn't find a way to make the window borderless.

How to create window not have borderless?

My code:
```
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:vm="using:HKLauncher.ViewModels"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:views="clr-namespace:HKLauncher.Views"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="HKLauncher.Views.MainWindow"
	    ExtendClientAreaToDecorationsHint="True"
        ExtendClientAreaChromeHints="NoChrome"
        ExtendClientAreaTitleBarHeightHint="-1"
        Icon="/Assets/avalonia-logo.ico"
		CanResize = "False"
        Title="HKLauncher">

	    <Window.Background>
		    <ImageBrush Source="/Assets/launcher.png"/>
	    </Window.Background>
        <views:MainView />
</Window>
```

![image](https://github.com/AvaloniaUI/Avalonia/assets/9639276/eb4ced1e-24cb-4389-bc43-a231838ecaca)

Thank you

**Comment by timunie** [comment]
`SystemDecorations="None"`

**Comment by cardinalcolor** [comment]
Well, it doesn't work. After migration from Avalonia 10 to 11 same problem. Window not reacting to setting SystemDecoration="None"

> `<Window xmlns="https://github.com/avaloniaui"
> 			xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
> 			xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
> 			xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
> 			mc:Ignorable="d" d:DesignWidth="455" d:DesignHeight="670"
> 			CanResize="False"
> 			Background="Transparent"
> 		BorderBrush="Transparent"
> 		BorderThickness="0"
> 			TransparencyLevelHint="Transparent"
> 		SystemDecorations="None"
> 		ExtendClientAreaToDecorationsHint="True"
>         ExtendClientAreaChromeHints="NoChrome"
>         ExtendClientAreaTitleBarHeightHint="-1"
> 			ShowInTaskbar="True"
> 			Width="445"
> 			Height="670"
> 			Icon="
> /Assets/icons/icon_windows.ico"
> 			FontFamily="{StaticResource MontserratFont}">`

![Screenshot 2024-04-15 113029](https://github.com/AvaloniaUI/Avalonia/assets/138114570/edc845f0-1941-41ea-be5e-759e2a1cc30e)


"Avalonia" Version="11.0.10"

**Reply by thevortexcloud** [reply]
Did it work in .10?

**Reply by cardinalcolor** [reply]
Yes, it worked before migration ("Avalonia" Version="0.10.22")

**Reply by timunie** [reply]
please check 11.1 beta as well

**Reply by cardinalcolor** [reply]
Still the same

![Screenshot 2024-04-15 140835](https://github.com/AvaloniaUI/Avalonia/assets/138114570/04499af9-d905-43de-ab4a-f9516d70f821)

**Comment by Sion10032** [comment]
For anyone who runs into the same issue, as I mentioned [here](https://github.com/AvaloniaUI/Avalonia/discussions/20015#discussioncomment-15356022):
when `SystemDecorations = SystemDecorations.None;` is used together with `ExtendClientAreaToDecorationsHint = true;`, a window border appears. When I remove `ExtendClientAreaToDecorationsHint = true;`, everything works as expected.
